microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -114,34 +102,30 @@ The iOS simulator for Mac OS X comes with Xcode, which you [install from the App
 
 When you are done, learn how to work with the simulator through [Apple Developer documentation](https://help.apple.com/simulator/mac/current).
 
-> [!NOTE]
-> To avoid having to open Xcode every time you want to use the iOS Simulator, open <!--Xcode, or iOS Simulator?-->it, right-click the iOS Simulator icon in your dock, and then select **Keep in Dock**.  Now just click the icon whenever you need it.
+To avoid having to open Xcode every time you want to use the iOS Simulator, open the iOS Simulator, right-click the iOS Simulator icon in your dock, and then select **Keep in Dock**.  Now just click the icon whenever you need it.
```

Comment:
```suggestion
To avoid having to open Xcode every time you want to use the iOS Simulator, open the iOS Simulator, right-click the iOS Simulator icon in your dock, and then select **Keep in Dock**.  Now you can just click the icon whenever you need it.
```